tests/unit/amo/components/TestSearchResults.js

```diff
@@ -11,6 +11,7 @@ import SearchResults from 'amo/components/SearchResults';
 import I18nProvider from 'core/i18n/Provider';
 import { fakeAddon } from 'tests/unit/amo/helpers';
 import { fakeI18n } from 'tests/unit/helpers';
+import { SEARCH_SORT_UPDATED } from 'core/constants';
 
 
 describe('<SearchResults />', () => {
```

Comment:
This is a pretty old test suite -- while you're in here could you make it `describe(__filename, ...)`? We have [this issue](https://github.com/mozilla/addons-frontend/issues/2928) to fix them all but it will be a lot of work.